imports/state/epics/resend-verification.js

```diff
@@ -0,0 +1,28 @@
+import { Accounts } from 'meteor/accounts-base'
+import { Meteor } from 'meteor/meteor'
+import { Subject } from 'rxjs/Subject'
+import { RESEND_VERIFICATION, ERROR, SUCCESS } from '../../ui/unverified-warning/unverified-warning.actions'
+
+import 'rxjs/add/operator/switchMap'
+import 'rxjs/add/operator/filter'
+
+export const resendVerification = action$ =>
+  action$.ofType(RESEND_VERIFICATION)
+    .filter(() => !Meteor.userId())
+    .switchMap(({userId}) => {
+      const meteorResult$ = new Subject()
+      Accounts.sendVerificationEmail({userId}, err => {
```

Comment:
I just noticed that "sendVerificationEmail" can only be called in Meteor's server side code (this is client side). You should define a new meteor "method" in *custom-users.js* and in it call `Accounts.sendVerificationEmail`.
Btw, it receives the user's id as the first arg directly, you shouldn't wrap it in curly brackets